docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,63 +14,58 @@ endif::[]
-We refer these two methods as "provisioning with shared credentials" and "provisioning with device credentials".
 
-* *Provisioning with shared credentials*
-+
-This type of provisioning is great for testing because the OTA Connect server plays the role of both the "issuer" and "verifier" when creating device certificates. You can try out OTA Connect without involving the rest of your infrastructure. However, it's not secure enough for a production scenario.
-+
-* *Provisioning with device credentials*
-+
-This is the provisioning method that you should use in production. In this scenario, the OTA Connect server doesn't issue any credentials to devices; it simply inspects device certificates that come preinstalled. The OTA Connect server only plays the role of "verifier" and the "issuer" role is handled by your fleet certificate authority.
+OTA Connect supports two provisioning methods. These methods determine the components that will issue and verify the certificate in your infrastructure. We refer to these two methods as "*provisioning with shared credentials*" and "*provisioning with device credentials*". The end result of these two provisioning methods is the same, as they both provide a device with a signed certificate that can be used for communication.
+
+
+For secure communication, a vehicle needs to have an X.509 certificate and it needs to trust the device gateway’s X.509 certificate. (Each account has its own unique device gateway, and that certificate is included in the credentials.zip and baked into the image as a pinned CA when you bitbake.) The device gateway trusts a vehicle’s X.509 certificate if and only if that certificate is signed by what we call the fleet root certificate authority (CA). Every device, regardless of how it is initially provisioned, ends up with its own unique X.509 certificate signed by the fleet root CA. Whenever the device interacts with the OTA Connect server, the device gateway authenticates and identifies it by its X.509 certificate.
+
+
+Note that in both cases, the server must have access to the fleet root CA for your fleet. The fleet root CA is used to sign the certificate of the connected devices.
 
-.How the server roles change depending on which method you choose:
-[caption="Figure 1: "]
-image::prov-diff-infra.png[]
+Let's take a closer look at how each method works.
 
 
-Note that in both cases, the server that plays "issuer" role, must have access to the private key and the root certificate for your fleet. The root certificate and private key are used to sign identity metadata and to verify the identity of connected devices.
+=== Provisioning with shared credentials
 
-Let's take a close look at how each method works.
+This type of provisioning is great for testing because the OTA Connect server manages the certificates and keys of the device. You can try out OTA Connect without involving the rest of your infrastructure. However, this method is not secure enough for a production scenario.
```

Comment:
Technically, the server doesn't "manage" the "keys of the device", although it does generate them. Not sure that detail is significant, but it may help to be clear.

Also, I'd prefer "this method may not be secure enough". There are more risks with this scenario, but a user may accept them if they are appropriately mitigated.